app/assets/stylesheets/tags.scss

```diff
@@ -0,0 +1,27 @@
+.tag-list {
+  margin-top: 2rem;
+}
+
+.tags {
+  margin-bottom: 0.8rem;
+}
+
+.tag {
+  @extend .chip;
+  background-color: #E6E18F;
```

Comment:
Color literals like `#E6E18F` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E6E18F` should be written as `#e6e18f`